doc/book/container/factories.md

```diff
@@ -37,78 +37,78 @@ order to seed the `Application` instance:
 
 - `middleware_pipeline` can be used to seed pre- and/or post-routing middleware:
 
-  ```php
-  'middleware_pipeline' => [
-      // An array of middleware to register prior to registration of the
-      // routing middleware:
-      'pre_routing' => [
-      ],
-      // An array of middleware to register after registration of the
-      // routing middleware:
-      'post_routing' => [
-      ],
```

Comment:
I actually want to keep the original formatting here. While we're currently publishing to RTD, which will not render this correctly, our plan is to use [bookdown](http://bookdown.io) to publish docs on the ZF website. Bookdown uses CommonMark, which correctly interprets this as a code block, and, further, as a code block associated with the preceding bullet point, which is the intent.